package-lock.json

```diff
@@ -344,18 +347,6 @@
             "integrity": "sha1-DELU+xcWDVqa8eSEus4cZpIsGyE=",
             "dev": true
         },
-        "node_modules/ansi-colors": {
```

Comment:
I dont think these changes are needed other tan bumping version number in package-lock.json. Please revert these changes

Reply:
Haven't made any change manually.. just did a npm install and npm build:pord, and it generated this loc file, since we will be deploying the results of npm build:prod to the extension store, i think it makes sense to push all the changes here, what do you say?